ex9/js/ajaxGet.js

```diff
@@ -0,0 +1,23 @@
+define(function() {
+    return function(url, callback) {
+      //url = '';
+      $.ajax({
+        type: 'GET',
+        url: url,
+        statusCode: {
+          404: function() {
+            alert( "page not found" );
```

Comment:
I wouldn't do this error specific approach. There are MANY errors that can occur. The error callback is enough